### ISSUE TITLE
electron + vuew + vue-extend-layout

### ISSUE THREAD

**Issue by apavelm** [ktquez/vue-extend-layout#11]
Hello!
I'm making an electron application with vuejs 2 and webpack4.
I am trying to use vue-extend-layout component for multiple layouts but faced with an error
> Uncaught TypeError: require.context is not a function
from this line
> var a = require.context('@/layouts', !1, /^\.\/.*\.vue$/);
I tried to create "/layouts" folder in different places without luck.

**Comment by apavelm** [comment]
Little extension, but it doesn't solve described problem in full
[src.zip](https://github.com/ktquez/vue-extend-layout/files/2085085/src.zip)

**Comment by ktquez** [comment]
@apavelm 
Sorry for the delay in responding.

You config alias @ in webpack?
https://github.com/ktquez/vue-extend-layout#webpack

**Comment by apavelm** [comment]
Yes, @ is used by electron in webpack. I have double checked it.

**Comment by ktquez** [comment]
Check update [v1.1.3](https://github.com/ktquez/vue-extend-layout/tree/v1.1.3)

But that may be some incompatibility with the electron.

Can you show me how the directory structure of your application is?

**Comment by apavelm** [comment]
directory structure was standard for electron + vuejs project
/main
/renderer - <dir where vue>
/static

I solved this by writing own component inspired by yours :-)
I suspect some incompatibility because of I tried to debug this situation by placing console.log() for paths, and everything looked fine... including @
